src/MonsterActor.cpp

```diff
@@ -0,0 +1,33 @@
+#include "MonsterActor.h"
+
+
+MonsterActor::MonsterActor(){
+  Position position = Position();
```

Comment:
do what Jack did with his branch and add init() and remove the position and velocity types